hugegraph-core/src/main/java/org/apache/hugegraph/backend/query/Query.java

```diff
@@ -50,7 +50,7 @@ public class Query implements Cloneable {
     public static final long NO_CAPACITY = -1L;
     public static final long DEFAULT_CAPACITY = 800000L; // HugeGraph-777
 
-    private static final ThreadLocal<Long> CAPACITY_CONTEXT = new ThreadLocal<>();
+    protected static final ThreadLocal<Long> CAPACITY_CONTEXT = new ThreadLocal<>();
```

Comment:
it does not expect to be accessed by others classes

Reply:
fixed